ceexon/the-Questioner
# Ft ui meetups main 162982794

# Main Meetup UI page

**What this PR entails?**
UI template for the page a user is redirected after logging in

**Description of tasks completed**
- a user can views most recent meetups
- create a page for the user after logging in
- user is able to access their profile

**How to manually test it**
- Ensure you have internet
- Open your browser  
- click on this link 
 ```
       https://kburudi.github.io/the-Questioner/UI/templates/meetups.html
```
- You will access the page

 **Related PT story**
#162982794


![recent-main](https://user-images.githubusercontent.com/45054246/50693420-f0c9f300-1047-11e9-8058-2f023ad97515.png)
![footer-main-user](https://user-images.githubusercontent.com/45054246/50693421-f1628980-1047-11e9-850b-a642b49f62fc.png)
